usefulUtilities/scriptPauser/scriptPauser.js

```diff
@@ -0,0 +1,102 @@
+//
+//  scriptPauser.js
+//
+//  Created by Zach Fox on 2019-02-26
+//  Copyright 2019 High Fidelity, Inc.
+//
+//  See accompanying README.md for usage instructions.
+// 
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+
+
+(function () {
+    // The names of scripts to pause.
+    // Populated from userData.
+    // The script names in the `fuzzyMatch` array can be partial script names.
+    // That makes using fuzzy matching MORE DANGEROUS!
+    // The script names in the `exactMatch` array must match the script to be paused exactly.
+    var scriptsToStopFuzzyMatch = [];
+    var scriptsToStopExactMatch = [];
+    // This is a list of apps that this entity script explicitly stopped.
+    // The scripts in this list will be restarted once this entity script is unloaded.
+    var pausedAppList = [];
+
+    var ScriptPauser = function() {};
+
+    ScriptPauser.prototype = {
```

Comment:
Do we need 2 lines between function definitions?

Reply:
yes, thank you